tests/pytests/integration/runners/test_nacl.py

```diff
@@ -0,0 +1,165 @@
+"""
+Tests for the nacl runner
+"""
+
+import pytest
+
+import salt.config
+import salt.utils.stringutils
+from tests.support.mock import patch
+
+pytest.importorskip("nacl.public")
+pytest.importorskip("nacl.secret")
```

Comment:
decide to test for the exact ones that were being imported, just in case.  I understand 'nacl' should do it, but what if something changed inside pynacl, this would catch it. Just a matter of caution.

Reply:
Ok. It's fine.
Just remove the commits changing pyyaml and cython and rebase.